docs/source/introduction/examples_output.rst

```diff
@@ -1,8 +0,0 @@
-.. dropdown:: Distribution Statement
-
- | # # # This source code is protected under the license referenced at
- | # # # https://github.com/NRLMMD-GEOIPS.
-
-*********************
```

Comment:
Will this be automated with the CLI?

Reply:
? This file was removed since it was blank

Reply:
I mean will producing example outputs for the documentation (probably from the test scripts) be automated with the CLI?  These documents were placeholders for things we needed to add, just want to make sure we are still tracking that they need to be added.

Reply:
(And that's what I meant with all my other "will this be automated with the CLI" comments as well, on other files)

Reply:
^ I think they definitely should be automated via the CLI. I do that with brassy, and it works well. I talked with @jsolbrig a bit about it, and I think the main concern is the time it might take to generate multiple products just to build the "final" docs

Reply:
As for this PR, I'm hoping to remove blank sections and create issues from them. That way we continue to track them but they're not empty pages on the user-facing docs

Reply:
I actually think I know how we can do this relatively quickly. I think we could set up individual jobs for each script. We could use the `matrix` method in github actions to generate jobs for a bunch of the test scripts, then let them run in individual jobs. They’d be relatively quick that way. We’d want to run those on a self-hosted runner so that the data are available, though.

Reply:
Created #721 for this.